querydsl-core/src/test/java/com/mysema/query/DefaultQueryMetadataTest.java

```diff
@@ -94,6 +95,20 @@ public void GetJoins2() {
     }
 
     @Test
+    public void GetJoins3() {
```

Comment:
I agree, but this is a backport, in general we do only fixes in Querydsl 3, and not stylistic improvements